packages/components/togglebutton/docs/spec.md

```
@@ -0,0 +1,154 @@
+# Overview
+
+The ToggleButton component is a UI element that allows users to switch between two states, typically "on" and "off". It operates similarly to a checkbox or switch, in that it can be toggled between two states.
+
+### Checkbox Vs Switch Vs ToggleButton
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)